File: musixise-box-api/src/main/java/com/musixise/musixisebox/api/web/vo/req/wechat/UnifiedorderVO.java
package com.musixise.musixisebox.api.web.vo.req.wechat;

import io.swagger.annotations.ApiModelProperty;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

public class UnifiedorderVO {

    @NotNull
    @Size(min = 1, max = 50)
    @ApiModelProperty(value = "商品ID", example = "商品ID")
    private Long productId;

    public Long getProductId() {
        return productId;
    }

    public void setProductId(Long productId) {
        this.productId = productId;
    }
}

File: musixise-box-shop/src/main/kotlin/com/musixise/musixisebox/shop/rest/web/vo/req/pay/UnifiedorderVO.kt
package com.musixise.musixisebox.shop.rest.web.vo.req.pay

import io.swagger.annotations.ApiModelProperty
import javax.validation.constraints.Size

class UnifiedorderVO {

    @Size(min = 1, max = 50)
    @ApiModelProperty(value = "支付订单ID", example = "20190106222026003b9ac9ff02540be3ff")
    var orderId: String = ""
}